pygmt/datasets/earth_relief.py

```diff
@@ -20,13 +28,15 @@ def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm
     this function. Afterwards, it will load the grid from the data directory.
```

Comment:
Line 19: if `data_source="gebco", the data will be downloaded to `~/.gmt/server/earth/earth_gebco/` instead. I think we can rewrite the sentence to something like "The grids are downloaded to a user data directory (usually a subdirectory under ``~/.gmt/server/``)"